doc/index.rst

```diff
@@ -22,10 +22,12 @@ In the third step fields are associated with each other.
 If one of the two associated messages has fields which is are not part of the other message they are being ignored.
 If both messages have fields the other message does not contain it is assumed that the mapping is incomplete and no association is established.
 
+If the package name, message name, and all field names and types are the same between a ROS 1 package and a ROS 2 package, then the messages will automatically be associated with each other without additional specification. The bridge needs to be recompiled, after the ROS 1 and ROS 2 packages have been compiled and sourced, each in a separate workspace, for a total of three workspaces.
```

Comment:
Please add a note that this is only true for package names with a certain suffix.